packages/language-support/src/tests/autocompletion/schemaBasedPatternCompletion.test.ts

```diff
@@ -64,6 +64,45 @@ describe('completeRelationshipType', () => {
     });
   });
 
+  test('Simple node completion pattern', () => {
+    const query = 'MATCH (t:Trainer)-[r:CATCHES]->(:';
```

Comment:
It'd be nice to add a few more test cases (longer pattern, different relationships). Kind of like we have for the rel types?